nautobot/extras/views.py

```diff
@@ -2118,6 +2118,17 @@ def get_extra_context(self, request, instance):
                 RequestConfig(request, paginate).configure(device_table)
                 context["device_table"] = device_table
 
+            if ContentType.objects.get_for_model(Interface) in context["content_types"]:
+                interfaces = instance.interfaces.select_related(
+                    "device",
+                    "status",
+                    "role",
+                ).restrict(request.user, "view")
+                interface_table = InterfaceTable(interfaces)
+                interface_table.columns.hide("role")
+                RequestConfig(request, paginate).configure(interface_table)
```

Comment:
Does each table need its own `paginate` dict? I see we're doing that for other views with multiple tables like `ConfigContextSchemaObjectValidationView`

Reply:
I feel like the `paginate` dicts in ConfigContextSchemaObjectValidationView are redundant? It all seems like the same dictionary to me. `get_paginate_count` gets the "per_page" data from request.GET which can only be one single unique value right?

Reply:
Yeah it is a bug, that behavior are confirmed on both `RoleRetrieveView` and `ConfigContextSchemaObjectValidationView`. Will submit an issue

Reply:
Not sure how we'd support different pagination on different tables within a single view - I guess we'd have to use separate query params for each table?